.github/ISSUE_TEMPLATE/feature-request.md

```diff
@@ -1,25 +1,18 @@
 ---
 name: Feature Request
-about: tell us about a new capability you want to see
+about: tell us about a new feature you want
 title: 
 labels: new feature
```

Comment:
We should clarify the use of new feature vs wishlist. I've always handled new feature as a tag for PRs and wishlist for something users would like to have (i.e. feature requests). We can rename wishlist to feature request, but eight now we have two tags with unclear connotation.

Reply:
I prefer one tag, new feature. Now github clearly links issues to PRs in the right hand block.

Reply:
Then we should get rid of wishlist. Though I still feel like new feature sounds a bit too demanding or definitive to me, whereas feature request wouldn't fit for PRs.

Reply:
If its just a feeling than you can overcome it with some meditation. :wink: